src/App.tsx

```diff
@@ -1,14 +1,115 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import * as todoService from './api/todos';
+import * as servises from './servises/buttons';
+import { ButtonProp } from './types/Button';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todoTitle, setTodoTitle] = useState('');
+  const [appliedTitle, setAppliedTitle] = useState('');
+  // const [todo, setTodo] = useState<Todo | null>(null);
+  const [cheked, setChecked] = useState(false);
+  const [loadContent, setLoadedContent] = useState<Todo[]>([]);
+  const [errorMessege, setErrorMessege] = useState('');
+  const [filteredBy, setFilteredBy] = useState<string>('All');
+
+  useEffect(() => {
+    todoService
+      .getTodos()
+      .then(response => {
+        const initData = response;
+
+        if (response.length === 0) {
+          setErrorMessege('Not found any todos');
+        }
+
+        setLoadedContent(initData);
+      })
+      .catch(error => {
+        setErrorMessege('Unable to load todos');
```

Comment:
error message should be hidden in 3 seconds.
Looks like it is a reason of failed test.
![Screenshot 2025-06-10 at 15 02 10](https://github.com/user-attachments/assets/8702ea8b-76d9-44ca-9fef-17ac5eebba4a)